tests/e2e/specs/wcpay/merchant/merchant-orders-status-change.spec.js

```diff
@@ -0,0 +1,235 @@
+			await page.waitForNavigation( {
+				waitUntil: 'networkidle0',
+			} );
+
+			// Verify the order status is set to processing.
+			const updatedOrderStatus = await page.$(
+				selectedOrderStatusSelector
+			);
+			await expect(
+				updatedOrderStatus.evaluate( ( el ) => el.innerText )
+			).resolves.toBe( 'Cancelled' );
+		} );
+	} );
+
+	describe( 'Change Status of order to Refunded', () => {
+		beforeAll( async () => {
+			// Place an order to change its status later
+			await setupProductCheckout(
+				config.get( 'addresses.customer.billing' )
+			);
+			const card = config.get( 'cards.basic' );
+			await fillCardDetails( page, card );
+			await shopper.placeOrder();
+			await expect( page ).toMatch( 'Order received' );
+
+			// Get the order ID so we can open it in the merchant view
+			const orderIdField = await page.$( orderIdSelector );
+			orderId = await orderIdField.evaluate( ( el ) => el.innerText );
+
+			// Login and open the order
+			await merchant.login();
+			await merchant.goToOrder( orderId );
+		} );
+
+		afterAll( async () => {
+			await merchant.logout();
+		} );
+
+		it( 'Show Refund Confirmation modal, do not change status if Cancel clicked', async () => {
+			// Select refunded from the order status dropdown.
+			await expect( page ).toSelect(
+				orderStatusDropdownSelector,
+				'Refunded'
+			);
+
+			// Verify the confirmation modal shows.
+			await page.waitForSelector( refundModalSelector, {
+				visible: true,
+			} );
+
+			// Click on Cancel.
+			await expect( page ).toClick( refundCancelSelector );
+
+			// Verify the order status is set to processing.
+			const selectedOrderStatus = await page.$(
+				selectedOrderStatusSelector
+			);
+			await expect(
+				selectedOrderStatus.evaluate( ( el ) => el.innerText )
+			).resolves.toBe( 'Processing' );
+
+			//click Update Order button and wait for page reloaded.
+			await expect( page ).toClick( '.save_order' );
+			await page.waitForNavigation( {
+				waitUntil: 'networkidle0',
+			} );
+
+			// Verify the order status is set to processing.
+			const updatedOrderStatus = await page.$(
+				selectedOrderStatusSelector
+			);
+			await expect(
+				updatedOrderStatus.evaluate( ( el ) => el.innerText )
+			).resolves.toBe( 'Processing' );
+		} );
+
+		it( 'Show Refund Confirmation modal, process Refund if confirmed', async () => {
+			// Select refunded from the order status dropdown.
+			await expect( page ).toSelect(
+				orderStatusDropdownSelector,
+				'Refunded'
+			);
+
+			// Verify the confirmation modal shows.
+			await page.waitForSelector( refundModalSelector, {
+				visible: true,
+			} );
+
+			// Click on Refund order.
+			await expect( page ).toClick( refundConfirmSelector );
+
+			// Wait for refund to be processed
+			await uiUnblocked();
+			await page.waitForNavigation( { waitUntil: 'networkidle0' } );
+
+			// Get the order price
+			const priceElement = await page.$( '.woocommerce-Price-amount' );
```

Comment:
Running `document.querySelectorAll('.woocommerce-Price-amount')` on any wp-admin order page, it will return multiple NodeList/Element(s) so I think we might choose a more specific selector, prefer a unique ID or a combination with it rather than a class name.

```suggestion
			const priceElement = await page.$( '#woocommerce-order-items .total .woocommerce-Price-amount' );
```

Reply:
Changed